lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
 // lms - views - shopping cart
 // ====================
 
+$enrollment-details-width-difference: $baseline*11.85;
```

Comment:
Sandbox: http://pr7987.m.sandbox.edx.org/
User `honor@example.com` has a paid course in the shopping cart.
SOL-837 is associated only with the first page of the payment flow.
Paid course is the one named "Paid Course"